### DISCUSSION TITLE
Option to NOT download accompanying image with each episode

### DISCUSSION THREAD

**Discussion by shooftie** [SamTV12345/PodFetch#425] [Ideas]
Most of the podcasts I listen to do not have separate images for each episode.

It would be really helpful to be able to choose whether or not to download the image alongside the episode so as to avoid wasted space.

<img width="908" alt="image" src="https://github.com/SamTV12345/PodFetch/assets/2161844/b3acfc31-e8ff-4399-b15c-6d04dba58743">

**Comment by PassiveLemon** [comment]
I would like this too, especially in a flat directory structure like shown above.
